files/zh-cn/web/html/element/input/index.html

```diff
@@ -472,15 +515,15 @@ <h3 id="属性各论">属性各论</h3>
  <dt>{{ htmlattrdef("accesskey") }} {{Deprecated_Inline}}</dt>
  <dd>用户按下后可以获得此控件焦点的单个字符。这是 HTML5 全局属性。</dd>
  <dt>{{ htmlattrdef("autocomplete") }}</dt>
- <dd>这个属性表示这个控件的值是否可被浏览器自动填充。如果<strong>type</strong>属性的值是 hidden、checkbox、radio、file，或为按钮类型（button、submit、reset、image），则本属性被忽略。可用的值是：
+ <dd>这个属性表示这个控件的值是否可被浏览器自动填充。如果 <strong>type</strong> 属性的值是 hidden、checkbox、radio、file，或为按钮类型（button、submit、reset、image），则本属性被忽略。可用的值是：
  <ul>
   <li><code>off</code>: 用户必须手动填值，或者该页面提供了自己的自动补全方法。浏览器不对此字段自动填充。</li>
   <li><code>on</code>: 浏览器可以根据用户先前的填表情况对此字段自动填值。</li>
   <li><code>name</code>: 完整的姓名</li>
-  <li><code>honorific-prefix: </code>Prefix or title (e.g. "Mr.", "Ms.", "Dr.", "Mlle")</li>
-  <li><code>given-name </code>：名</li>
+  <li><code>honorific-prefix</code>: Prefix or title (e.g. "Mr.", "Ms.", "Dr.", "Mlle")</li>
+  <li><code>given-name</code>: 名</li>
```

Comment:
See <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#标点符号>, please use `：` instead.

```suggestion
  <li><code>given-name</code>：名</li>
```